canonical-websites/snapcraft.io
# Move tests to have same shape as webapp

# Summary

Shape tests so that they have the same architecture as templates and webapp

# QA

- On master
- `./run test`
- check the number of python tests
- checkout on this branch
- `./run test`
- the number of tests should be the same (and pass lol)
